src/debugging/Inspector/scripts/generate_typescript_interfaces.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import sys
+import getopt
+
+def main(argv):
+    usage = 'Usage: generate_typescript_interfaces.py --inspector-scripts-path <path to JavaScriptCore/inspector/scripts> --combined-domains-path <path to CombinedDomains.json> --output-dir <result directory path>'
```

Comment:
I have a question - can you please explain to me where these arguments come from and how do they look.